types/src/legacy_sse_data/translate_deploy_hashes.rs

```diff
@@ -0,0 +1,37 @@
+use casper_types::{BlockBodyV2, DeployHash, TransactionHash};
+use mockall::automock;
+
+#[automock]
+pub trait DeployHashTranslator {
+    fn translate(&self, block_body_v2: &BlockBodyV2) -> Vec<DeployHash>;
+}
+
+#[derive(Default)]
+pub struct DefaultDeployHashTranslator;
+
+#[derive(Default)]
+pub struct TransferDeployHashTranslator;
+
+impl DeployHashTranslator for DefaultDeployHashTranslator {
+    fn translate(&self, block_body_v2: &casper_types::BlockBodyV2) -> Vec<DeployHash> {
+        block_body_v2
+            .standard()
```

Comment:
It seems like legacy deploy can also land in the "mint" bucket. See [here](https://github.com/casper-network/casper-node/blob/feat-2.0/types/src/transaction/deploy.rs#L1150-L1158). Could be worth double-checking if we can handle this in the translation process.